app/setup.py

```diff
@@ -324,6 +327,22 @@ def setup_submitter(application):
         raise Exception("Unknown EQ_SUBMISSION_BACKEND")
 
 
+def setup_publisher(application):
+    if application.config["EQ_PUBLISHER_BACKEND"] == "pubsub":
+        topic_id = application.config.get("EQ_PUB_SUB_TOPIC_ID")
+
+        if not topic_id:
+            raise Exception("Setting EQ_PUB_SUB_TOPIC_ID Missing")
```

Comment:
Isn't it just "EQ_PUB_SUB_TOPIC_ID Missing" as we have not set anything yet?

Reply:
It is saying "The application setting eq_pub_sub_topic_id is missing" and not that "we are setting something". I followed the existing pattern used in runner but happy to change this and existing ones as I can see where the confusion came from.